packages/now-cli/test/helpers/prepare.js

```diff
@@ -245,6 +245,11 @@ module.exports = (req, res) => {
     'static-deployment': {
       'index.txt': 'Hello World',
     },
+    nowignore: {
+      'index.txt': 'Hello World',
+      'ignore.txt': 'Should be ignored',
+      '.nowignore': 'ignore.txt',
+    },
```

Comment:
I think we need a second fixture with the wildcard allowlist so that the first line is `*`

https://github.com/styfle/packagephobia/blob/master/.nowignore

Reply:
```
'nowignore-wildcard': {
      'index.txt': 'Hello World',
      'ignore.txt': 'Should be ignored',
      '.nowignore': '*\n!index.txt',
    }
```